edx_sga/static/js/src/edx_sga.js

```diff
@@ -12,6 +12,7 @@ function StaffGradedAssignmentXBlock(runtime, element) {
         var removeGradeUrl = runtime.handlerUrl(element, 'remove_grade');
         var template = _.template($(element).find("#sga-tmpl").text());
         var gradingTemplate;
+        var maxFileUploadSize = 3998272;
```

Comment:
This actually should be grabbed from the django file storage settings so that it can be changed per instance, e.g: https://github.com/edx/edx-platform/blob/master/lms/envs/aws.py#L446